src/dot.cpp

```diff
@@ -397,8 +397,7 @@ static bool convertMapFile(FTextStream &t,const char *mapName,
       {
 	QCString replBuf = replaceRef(buf,relPath,urlOnly,context);
         int indexS = replBuf.find("id=\""), indexE;
-        indexE=replBuf.find('"',indexS+4);
-        if (indexS>=0 && (indexE=buf.find('"',indexS))!=-1)
+        if (indexS>=0 && (indexE=replBuf.find('"',indexS+4))!=-1)
```

Comment:
I guess `indexS` has to be bigger than 0 otherwise `left(indexS-1)` will refer to a negative size

Reply:
In principle yes, but in this case it is OK as the line `if (buf.left(5)=="<area")` already gives a length > 0. when the find is executed.